src/components/text-area/text-area.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ChangeEvent, useState } from 'react';
+import { ReactComponent as LockIcon } from '../../assets/svg/security-lock.svg';
+import './text-area.scss';
+
+export interface TextAreaProps {
+  lines: number;
+  initialText?: string;
+  label?: React.ReactNode;
+  placeholder?: string;
+  readonly?: boolean;
+  resizable?: boolean; // default: false
+  onChange?: (value: string) => void;
+}
+
+export const TextArea = ({
+  lines,
+  initialText,
+  label,
+  placeholder,
+  readonly,
+  resizable,
+  onChange,
+}: TextAreaProps) => {
+  const [value, setValue] = useState(initialText);
```

Comment:
Could we move this out of the component and up a level? So have a prop `value` instead `initialText`. With the import and export dialog, we will need to be able to set the text repeatedly

Reply:
Yeah, I didn't think about a way of doing that without using `useRef` but I think this makes sense now.